src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -58,4 +58,28 @@
       </attribute>
     </additionalAttributes>
   </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.IO.FilePath.HasExtension</oldName>
+    <newName>DSCore.ImportExport.File.FileHasExtension</newName>
```

Comment:
I would suggest keeping the names of the namespaces unchanged, only changing the class names as and when required.